apps/image_augmentation/image_augmentation.cpp

```diff
@@ -272,11 +274,14 @@ int main(int argc, const char ** argv)
         if(color_format ==  RocalImageColor::ROCAL_COLOR_RGB24 )
         {
             cv::cvtColor(mat_output, mat_color, CV_RGB2BGR);
-            cv::imshow("output",mat_color);
+            // cv::imshow("output",mat_color);
+            cv::imwrite("output.png",mat_color);
         }
         else
         {
-            cv::imshow("output",mat_output);
+            // cv::imshow("output",mat_output);
```

Comment:
why imshow removed?

Reply:
In dockers, imshow doesnt work. Changing it back